dashboard/app/models/concerns/text_to_speech.rb

```diff
@@ -0,0 +1,97 @@
+require 'acapela'
+require 'net/http'
+require 'uri'
+require 'redcarpet'
+require 'redcarpet/render_strip'
+
+TTS_BUCKET = 'cdo-tts'
+
+# american candidate
+
+# british candidate
+VOICES = {
+  rosie: {
+    VOICE: 'rosie22k',
+    SPEED: 180,
+    SHAPE: 100
+  },
+  ella: {
+    VOICE: 'ella22k',
+    SPEED: 140,
+    SHAPE: 98
+  },
+  sharon: {
+    VOICE: 'sharon22k',
+    SPEED: 180,
+    SHAPE: 100
+  }
+}
+
+module TextToSpeech
+  extend ActiveSupport::Concern
+
+  # TODO: this concern actually depends on the SerializedProperties
+  # concern ... I'm not sure how best to deal with that.
```

Comment:
Any thoughts on the appropriate way to deal with this? Should this concern include the SerializedProperties concern? Or perhaps extend it?